styles/widgets/generic/fileManager.generic.less

```diff
@@ -1,12 +1,87 @@
 .dx-filemanager {
+    border-color: @datagrid-border-color;
```

Comment:
Do not use constants related to other widgets. It may lead to unexpected results when our users will customize a theme via ThemeBuilder.
The better way - define new constants for all [color-schemes](https://github.com/DevExpress/DevExtreme/tree/19_1/styles/widgets/generic/color-schemes)

Reply:
Fixed